AxonDeepSeg/models/default_SEM_model_v1/checkpoint

```diff
@@ -1,4 +1,2 @@
 model_checkpoint_path: "model.ckpt"
-all_model_checkpoint_paths: "best_loss_model.ckpt"
```

Comment:
Why are we no longer saving the checkpoint for the best loss and best accuracy models?

Reply:
We are saving best loss and best accuracy in the` model` folder though its in `HDF5` format(keras save file in this format) as the earlier one was in `CKPT `format . I can convert these files to `CKPT` format if required and we can mention it in `checkpoint file`. One question: Do you know  if is there a need to keep ` best_loss_model.ckpt`, `best_acc_model.ckpt` in` checkpoint `file, I could find a different way to overcome this issue.

Reply:
Fixed in (fa7b63f)